packages/react/src/components/controls/ChatToggle.tsx

```diff
@@ -8,18 +8,23 @@ interface UseToggleChatProps {
 }
 
 function useToggleChat({ props }: UseToggleChatProps) {
-  const { dispatch } = useLayoutContext().widget;
+  const { dispatch, state } = useLayoutContext().widget;
   const { className } = React.useMemo(() => setupChatToggle(), []);
 
   const mergedProps = React.useMemo(
     () =>
-      mergeProps(props, {
-        className,
-        onClick: () => {
-          if (dispatch) dispatch({ msg: 'toggle_chat' });
+      mergeProps(
+        props,
+        {
+          className,
+          onClick: () => {
+            if (dispatch) dispatch({ msg: 'toggle_chat' });
+          },
         },
-      }),
-    [props, className, dispatch],
+        // @ts-expect-error custom data props
+        { 'data-lk-active': state?.showChat ? 'true' : 'false' },
```

Comment:
Any reasons for the new custom data attribute? Have you considered using the `aria-ressed` attribute like we do for the other control buttons?

Reply:
aria-pressed is a lot better, you're right!
We use `data-lk-active` in both `deviceSelector` and `paginationIndicator`. I guess at least for `deviceSelector` it should then also be using `aria-pressed` ?